graphql/types/ReturnRequest.graphql

```diff
@@ -187,6 +187,7 @@ input ReturnRequestFilters {
   sequenceNumber: String
   id: String
   createdIn: DateRangeInput
```

Comment:
Let's document it in the code. Please, add a similar comment into the line 73 (createdIn for the type ReturnRequestResponse)

```suggestion
"""
  createdIn: It uses the field dateSubmitted in the return request schema to search for documents.
  The field createdIn is auto generated when the document is created, not reflecting the real value for documents migrated from older versions.
  """
  createdIn: DateRangeInput
```